### ISSUE TITLE
Review of Acumos AI

### ISSUE THREAD

**Issue by cekbote** [cekbote/coredns_ml_plugin#9]
TLDR: It can be used for any machine learning project (theoretically), however, there is a long way for the tool to be good enough for people to adopt it. 

What is it?

From what I understand, it is a platform where users can use models and preprocessing functions created by other people by just dragging and dropping them into a canvas and then connecting them. Moreover, different people can create models for the same tasks and then the best one can be selected. The total setup feels like a spinoff of Kaggle but with a larger focus on deployment, Moreover, I think the best part is that while deploying, the model can be converted into a microservice by the platform itself.  

The Pros:
- Ease or prototyping and using other peoples code.
- While deploying, the model can be converted into a microservice by the platform itself.  
- Works very well for common tasks such as facial recognition.

The Cons:
- Doesn't work well for specialised tasks as you'd have to create the entire model on your own anyways. 
- Many models are unavailable on the platform that are the current state of the art. (Due to the lack of a lot of users)
- A lot of work has to be still done to make the platform more user friendly

Links:
- [The official homepage](https://lfai.foundation/projects/acumos-ai/)
- [General overview](https://docs.acumos.org/en/demeter/architecture/intro.html)
